app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1191,7 +1191,7 @@ class BrowserTabFragment :
         val binding = ContentSiteLocationPermissionDialogBinding.inflate(layoutInflater)
 
         val title = domain.websiteFromGeoLocationsApiOrigin()
-        binding.sitePermissionDialogTitle.text = title
+        binding.sitePermissionDialogTitle.text = getString(R.string.preciseLocationSiteDialogTitle, title, title)
```

Comment:
The string format just need one argument
```suggestion
        binding.sitePermissionDialogTitle.text = getString(R.string.preciseLocationSiteDialogTitle, title)
```